sdk/websites/Azure.ResourceManager.AppService/src/Customization/Models/AppServiceTableStorageApplicationLogsConfig.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.ResourceManager.AppService.Models
+{
+    public partial class AppServiceTableStorageApplicationLogsConfig
+    {
+        /// <summary> Uri of the resource. </summary>
+        [EditorBrowsableAttribute(EditorBrowsableState.Never)]
+        [ObsoleteAttribute("This property has been replaced by ResourceUriString")]
+        public System.Uri SasUri { get; set; }
+
+        [EditorBrowsableAttribute(EditorBrowsableState.Never)]
+        [ObsoleteAttribute("This property has been replaced by ResourceUriString")]
```

Comment:
Two issues here:
1. We should remove the `Attribute` suffix here. We usually do not append this suffix when using attributes
2. We should add `false` to `Obsolete` to make sure less impacts to our customers. This tells the compiler to generate warnings when using the obsoleted members instead of generating errors.
```csharp
[Obsolete("the message", false)]
```